src/newViews/ProjectEdit/ProjectDetailsForm/index.tsx

```diff
@@ -441,6 +547,10 @@ function ProjectDetailsForm(props: Props) {
                     </Button>
                 )}
             />
+            <Prompt
+                when={!pristine}
+                message={_ts('common', 'youHaveUnsavedChanges')}
+            />
```

Comment:
@subinasr , this much is okay for Prompt.

Reply:
Yeah, thanks bro!